taxi/urls.py

```diff
@@ -16,12 +22,30 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/create",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create",
```

Comment:
```suggestion
        "manufacturers/create/",
        ManufacturerCreateView.as_view(),
        name="manufacturer-create",
```
add trailing `/`
fix everywhere